test/new-e2e/system-probe/test-runner/main.go

```diff
@@ -193,13 +191,22 @@ func testPass(testConfig *testConfig, props map[string]string) error {
 	}
 
 	for _, testsuite := range testsuites {
-		pkg := pathToPackage(testsuite)
+		pkg, err := filepath.Rel(testConfig.testDirRoot, filepath.Dir(testsuite))
+		if err != nil {
+			return fmt.Errorf("could not get relative path for %s: %w", testsuite, err)
+		}
 		junitfilePrefix := strings.ReplaceAll(pkg, "/", "-")
 		xmlpath := filepath.Join(xmlDir, fmt.Sprintf("%s.xml", junitfilePrefix))
 		jsonpath := filepath.Join(jsonDir, fmt.Sprintf("%s.json", junitfilePrefix))
 		args := buildCommandArgs(pkg, xmlpath, jsonpath, testsuite, testConfig)
 
+		log.Printf("/go/bin/gotestsum %s", strings.Join(args, " "))
```

Comment:
Is there a reason you added this? I want to be sensitive to the test runner output and keep it high signal.

Reply:
Oops, leftover from some debugging.